catalog/catalog_products/views.py

```diff
@@ -0,0 +1,105 @@
+from typing import Dict
+from django.http.response import JsonResponse
+from rest_framework.generics import CreateAPIView
+from rest_framework.views import APIView
+from catalog_products.serializers import CategorySerializer, ProductSerializer
+from catalog_products.models import Category, Product
+from rest_framework import viewsets
+
+
+class CategoryAllViewSet(viewsets.ModelViewSet):
+    serializer_class = CategorySerializer
+    queryset = Category.objects.all()
+
+
+class ProductAllViewSet(viewsets.ModelViewSet):
+    serializer_class = ProductSerializer
+    queryset = Product.objects.all()
+
+
+class Task1ViewSet(APIView):
+    queryset = Category.objects.all()
+    serializer_class = CategorySerializer
+    def get(self, request):
+        category_all = Category.objects.all()
+        category_without_parent = []
+        category_with_parent = []
+        for category in category_all:
+            if category.parent != None:
```

Comment:
When dealing with None, True, False, please use `is` keyword. 

This is not the PEP 8 preferred approach to comparing values to None.

Best practice would be `if category.parent is not None:`